### DISCUSSION TITLE
Who is using this template?

### DISCUSSION THREAD

**Discussion by rubiin** [rubiin/ultimate-nest#514] [Show and tell]
Just a small survey . Hopefully everyone using will take their time to fill in .

**Comment by fac3m4n** [comment]
I played a bit with template. It looks really robust and planning on deploying it to prod. Just want to make sure about performance where we have 10k concurrent users

**Reply by rubiin** [reply]
I am working on a deployment script that should quickly set you up for development.Should be done in a week time
